frontend/src/index.tsx

```diff
@@ -14,6 +15,24 @@ if (process.env.NODE_ENV === 'test') {
   import('./mocks/msw/browser').then(({ worker }) => worker.start())
 }
 
+// Init Datadog RUM
+datadogRum.init({
+  applicationId: process.env.REACT_APP_DD_RUM_APP_ID || '',
+  clientToken: process.env.REACT_APP_DD_RUM_CLIENT_TOKEN || '',
+  env: process.env.REACT_APP_DD_RUM_ENV || '',
+  site: 'datadoghq.com',
+  service: 'formsg-react',
+
+  // Specify a version number to identify the deployed version of your application in Datadog
+  // version: '1.0.0',
+  sampleRate: 100,
+  replaySampleRate: 100,
```

Comment:
Reduce this to 10, 10. 

Also check if we can inject a version in the init call

Reply:
make the session sample be the reverse of react rollout percentage.